commands/orchestrationcommands/stackcommands/abandon.go

```diff
@@ -0,0 +1,146 @@
+package stackcommands
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/rackspace/rack/commandoptions"
+	"github.com/rackspace/rack/handler"
+	"github.com/rackspace/rack/internal/github.com/codegangsta/cli"
+	"github.com/rackspace/rack/internal/github.com/rackspace/gophercloud/rackspace/orchestration/v1/stacks"
+	"github.com/rackspace/rack/util"
+)
+
+var abandon = cli.Command{
+	Name:        "abandon",
+	Usage:       util.Usage(commandPrefix, "abandon", "[--name <stackName> | --id <stackID> | --stdin name]"),
+	Description: "Deletes an existing stack, but leave resources intact",
+	Action:      actionAbandon,
+	Flags:       commandoptions.CommandFlags(flagsAbandon, keysAbandon),
+	BashComplete: func(c *cli.Context) {
+		commandoptions.CompleteFlags(commandoptions.CommandFlags(flagsAbandon, keysAbandon))
+	},
+}
+
+func flagsAbandon() []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
+			Name:  "id",
+			Usage: "[optional; required if `stdin` or `name` isn't provided] The ID of the server.",
+		},
+		cli.StringFlag{
+			Name:  "name",
+			Usage: "[optional; required if `id` or `stdin` isn't provided] The name of the server.",
+		},
+		cli.StringFlag{
+			Name:  "stdin",
+			Usage: "[optional; required if `id` or `name` isn't provided] The field being piped into STDIN. Valid values are: name.",
+		},
+		cli.StringFlag{
+			Name:  "output-file",
```

Comment:
I don't think we need this. 2 reasons:
1. It's easy enough for users to redirect the output: `rack orchestration stack abandon --name myStack >> abandon.txt
2. It would apply to any command, and as such would be a global flag.